SU-SWS/stanford_media
# D8CORE-106: Multiple Media Formatter

# READY FOR (Initial) REVIEW 

# Summary
- Allows multiple options for an entity reference field referencing media entities.
![Screen Shot 2020-01-21 at 4 11 17 PM](https://user-images.githubusercontent.com/550602/72854273-b211d580-3c68-11ea-8b6c-55e8537d79bf.png)

# Review By (Date)
- Jan 29th

# Urgency
- Med/Low

# Steps to Test

1. Check out branch https://github.com/SU-SWS/acsf-cardinalsites/pull/51 or review on tugboat build
2. Review code and comment on the approach
3. Log in to the tugboat build and play around with the configuration options `/admin/structure/paragraphs_type/stanford_media_caption/display`

# ToDo
- Revise and add tests.

# Affected Projects or Products
- D8CORE
- Anything using media fields

# Associated Issues and/or People
- D8CORE
- 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
